ui/setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name='ecsdeploy',
-    version='2.11.0r',
+    version='2.9.1r',
```

Comment:
Why did we go from 2.11 to 2.9 here?

Reply:
there were some issues with 2.11 installer images , it was broken due to multiple package compatibility issues , so we had  to rolledback.